### PR TITLE
[storage] trivial: longer timeout in `Pruner::wake_and_wait()` (tests only)

### PR DESCRIPTION
**Pull request by msmouse**
## Motivation

Occationally this times out after code is just built.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

existing coverage

### CODE REVIEW

**Comment by metajack** [comment]
I'm going to test bors-libra with this PR, please don't hit the green button manually.

File: storage/libradb/src/pruner/mod.rs [hunk 1]
```diff
@@ -91,7 +91,7 @@ impl Pruner {
 
         if latest_version > self.num_historical_versions_to_keep {
             let least_readable_version = latest_version - self.num_historical_versions_to_keep;
-            const TIMEOUT: Duration = Duration::from_millis(100);
+            const TIMEOUT: Duration = Duration::from_secs(1);
```
Comment (zamsden) [inline storage/libradb/src/pruner/mod.rs]: I would consider a slightly longer timeout, or at least adding a comment here saying that this was a problem and could pop up again.

**Comment by metajack** [comment]
That looks like a real error, but bors should work now when that's fixed. The bot wasn't looking for the right check name.

**Comment by msmouse** [comment]
@zamsden  updated.

**Pull request closed without merging**